### DISCUSSION TITLE
Allow installs on servers without browser

### DISCUSSION THREAD

**Discussion by mews-se** [ThioJoe/YT-Spammer-Purge#1128] [Ideas]
It would be neat to be able to install this on a linux server without a browser. Anyway this is possible?

**Comment by 4yman-0** [comment]
>  this is a stand alone application that only needs the web browser for completing the authorisation

**Comment by 4yman-0** [comment]
Try setup from desktop and move files to server (if possible)

**Reply by Firecul** [reply]
Definitely the easiest way to get it working

**Comment by Firecul** [comment]
There has been some talk about this feature over time but no one seems to have taken up the challenge to my knowledge.
Links to other posts mentioning this topic: https://github.com/ThioJoe/YT-Spammer-Purge/discussions/205 https://github.com/ThioJoe/YT-Spammer-Purge/discussions/279 https://github.com/ThioJoe/YT-Spammer-Purge/discussions/491 https://github.com/ThioJoe/YT-Spammer-Purge/discussions/784 https://github.com/ThioJoe/YT-Spammer-Purge/discussions/929 https://github.com/ThioJoe/YT-Spammer-Purge/issues/1013
